src/lib/mina.ts

```diff
@@ -1039,3 +1044,134 @@ async function verifyAccountUpdate(
     );
   }
 }
+
+function verifyTransactionLimits(accountUpdates: AccountUpdate[]) {
+  // constants used to calculate cost of a transaction - originally defined in the genesis_constants file in the mina repo
+  const proofCost = 10.26;
+  const signedPairCost = 10.08;
+  const signedSingleCost = 9.14;
+  const costLimit = 69.45;
```

Comment:
I think we should eventually add this to auto-generation via the `snarky_js_constants.ml` mechanism, so it'd guaranteed to stay in sync. It's not urgent though and can be an issue for later